src/Sulu/Bundle/WebsiteBundle/Admin/WebsiteAdmin.php

```diff
@@ -44,4 +49,21 @@ public function getJsBundleName()
     {
         return 'suluwebsite';
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getSecurityContexts()
+    {
+        return [
+            'Sulu' => [
+                'Website' => [
+                    'sulu.website.cache' => [
```

Comment:
the cache will be completely cleared for all webspaces. /cc @chirimoya @webmozart can you add your thoughts about that?